MapboxDirections/MBVisualInstructionBanner.swift

```diff
@@ -21,6 +21,11 @@ open class VisualInstructionBanner: NSObject, NSSecureCoding {
      */
     @objc public let secondaryInstruction: VisualInstruction?
     
+    /**
+     The information about the next maneuver (the one after the upcoming maneuver). This detail gives the driver heads up when the current step is short.
+     */
+    @objc public let subInstruction: VisualInstruction?
```

Comment:
Rename this property to `subinstruction` (and elsewhere in this PR). Precedents for this include `NSCompoundPredicate.subpredicates`.

Reply:
@1ec5 I understand your preference for `subinstruction` based off, `NSCompoundPredicate.subpredicates`. In this context, it feels out of place when sub instruction variable reads differently from, `primaryInstruction` and `secondaryInstruction`. Probably, we should go for consistency here. 
https://github.com/mapbox/MapboxDirections.swift/pull/258/files#diff-8aa869dddbf0e52feb01bd7652980dfbL42

Reply:
“Sub” isn’t a word.

Reply:
@1ec5 okay, if that's your notion, then we're better off with a readable word like `tertiary` --> `tertiaryInstruction`. Which makes this consistent.